### PR TITLE
Css three column layout-David Gonzalez

### PR DESCRIPTION
**Pull request by David-Gonzalez11**
3 column

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey David, this does not match the example, please revise and submit a new screenshot with your revision.

**Comment by David-Gonzalez11** [comment]
3 column

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Please make sure to compare your output to the example and look for differences before submitting a new version
image

**Comment by David-Gonzalez11** [comment]
updated three column

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <img src="images/cloud-computing.png" />
+          </div>
+          <h1>Integrates with existing tools</h1>
+          <p>
+            Enzyme QMS integrates with your existing tools, such as JIRA, GitHub
+            and Jenkins with Enzyme
+          </p>
+        </div>
+
+        <div class="columnright">
```
Comment (thebearingedge) [inline css-three-column-layout/index.html:44]: Try changing your CSS so that you only need one column class instead of qms, middle, and right.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**